### PR TITLE
Add customer stories section to handbook

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Added a customer stories section to handbook

## Related Issue(s)

https://github.com/FlowFuse/website/pull/1478

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
@robmarcer this is it, please add what you see fit here.

File: src/handbook/customer/marketing/customer-stories.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+6. As a document reviewer, you should leave red-line edits in the document or add comments to the document.
+7. When the final version of the story is done, follow these steps to create a PDF version:
+    - Go to our [Template gallery](https://docs.google.com/document/u/0/?ftv=1&folder=1QXCso0HQPbShrB5_8N0n_sDnIlluUaaI&tgif=d) and select 'FlowFuse Customer Story - Template'.
+    - Change the file name by replacing 'Template' with the customer's name.
+    - Replace all the template information with your story. Ensure you paste using `ctrl + shift + v` to maintain the template format.
+    - Confirm that the file is saved in the [Google Docs version](https://drive.google.com/drive/folders/1SbZ7ISJaYcxcJ_00Y_CwdY5gd9_uOTHP?usp=drive_link) folder.
+    - Once ready, you can either print or download it as a PDF and then place it in the [PDFs folder](https://drive.google.com/drive/folders/1QXCso0HQPbShrB5_8N0n_sDnIlluUaaI?usp=drive_link).   
+8. Create a download form in HubSpot specifically for the PDF you just created:
+    - Replace with instructions
```
Comment (Yndira-FlowForge) [inline src/handbook/customer/marketing/customer-stories.md:26]: Hi @Hasmin-AC, could you please replace that line with the steps to create the form in HubSpot? Perhaps the last step could be 'Save the formId; you'll need it in the next steps,' explaining how to obtain it. What do you think?

Let me know if you need help editing this.

Comment (Hasmin-AC) [inline src/handbook/customer/marketing/customer-stories.md:26]:
```suggestion
    - Upload PDF file in Hubspot "Files and Templates" and copy the public URL
    - In Hubspot go to Marketing -> Lead Capture -> Forms and "Create Form" or clone a case study download form.  
    - Form should include first name, last name, email and company (not mandatory)
    - Add PDF url as the action after submission, under "Redirect to another page" 
    - Publish form
```

**Review by Hasmin-AC — CHANGES_REQUESTED** [review]

File: src/handbook/customer/marketing/customer-stories.md (hunk 1 above)
Comment (Yndira-FlowForge) [reply src/handbook/customer/marketing/customer-stories.md:26]: Could you please add a step saying how they can obtain the formId?

Comment (Hasmin-AC) [reply src/handbook/customer/marketing/customer-stories.md:26]:
```suggestion
     - Upload PDF file in Hubspot "Files and Templates" and copy the public URL
    - In Hubspot go to Marketing -> Lead Capture -> Forms and "Create Form" or clone a case study download form.  
    - Form should include first name, last name, email and company (not mandatory)
    - Add PDF url as the action after submission, under "Redirect to another page" 
    - Update and Publish form
    - Copy the formID from the script that is generated when you click on publish. 
```

**Review by Hasmin-AC — CHANGES_REQUESTED** [review]

**Comment by robmarcer** [comment]
@Yndira-FlowForge I've added some info about my methodology.

File: src/handbook/customer/marketing/customer-stories.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+---
+
+# Customer Stories
+
+## Creating a story
+
+A customer story is the same as a case study, these are helpful in illustrating how Node-RED and FlowFuse can be useful for our potential customers. The process would be as follows:
+
+1. Raise an issue for the content in our private [Github repository](https://github.com/FlowFuse/customer){rel="nofollow"}.
+2. If the content is date specific create an 'all day' event on the Google Calendar [FlowFuse Publishing Schedule](https://calendar.google.com/calendar/u/0?cid=Y18yMGFjMmM5MmMwYmE0YTYwNDg4NDE1MjBmMGU2YWE0MGFhZGUxNTlkNThjZGY0ZGMwMjA0NTI4ZjFjMTcxZmQ0QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20){rel="nofollow"}, include a link to the issue in the event description. If you don't have the permission to create events please ask our Google Workspace admin to give you access.
```
Comment (ZJvandeWeg) [inline src/handbook/customer/marketing/customer-stories.md:14]: I thought we stopped using that calendar a while ago in favour of https://github.com/orgs/FlowFuse/projects/18/views/1

cc @gdziuba 
```suggestion
2. If the content is date specific create an 'all day' event on the Google Calendar [FlowFuse Publishing Schedule](https://calendar.google.com/calendar/u/0?cid=Y18yMGFjMmM5MmMwYmE0YTYwNDg4NDE1MjBmMGU2YWE0MGFhZGUxNTlkNThjZGY0ZGMwMjA0NTI4ZjFjMTcxZmQ0QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20){rel="nofollow"}, include a link to the issue in the event description. If you don't have the permission to create events please ask our Google Workspace admin to give you access.
```

File: src/handbook/customer/marketing/customer-stories.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+3. If you want to work on the content immediately please triage the issue into our [development board](https://github.com/orgs/FlowFuse/projects/1/views/33).
+4. Conduct an interview call with the customer using Google Meet to record and transcribe the conversation.
+5. Consider that the readers of the story may have little knowledge of the industry discussed in the story. Try to avoid using language which excludes any potential FlowFuse customer from understanding and gaining value from the content.
```
Comment (ZJvandeWeg) [inline src/handbook/customer/marketing/customer-stories.md:17]:
```suggestion
5. Consider that the readers of the story may have little knowledge of the industry discussed in the story. Try to explain industry specific terms or avoid using language which excludes any potential FlowFuse customer from understanding altogether.
```

File: src/handbook/customer/marketing/customer-stories.md (hunk 2 above)
Comment (Yndira-FlowForge) [reply src/handbook/customer/marketing/customer-stories.md:14]: I'll remove it, then.

Comment (gdziuba) [reply src/handbook/customer/marketing/customer-stories.md:14]: Yes.  That is the plan that @Hasmin-AC and I came up with.
```suggestion
2. Please update and/or create an issue and schedule it within [Content Calender](https://github.com/orgs/FlowFuse/projects/21)
```

Comment (Yndira-FlowForge) [reply src/handbook/customer/marketing/customer-stories.md:14]: How about this? In the previous step the issue is created in the content repository.
```suggestion
2. Add it to the [Content Calendar Project](https://github.com/orgs/FlowFuse/projects/21) and schedule it.
```

File: src/handbook/customer/marketing/customer-stories.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
+---
+navTitle: Customer Stories
+meta:
+    title: Marketing - Customer Stories
```
Comment (ZJvandeWeg) [inline src/handbook/customer/marketing/customer-stories.md:4]: Why both? And maybe you should work with @joepavitt to understand why there's many ways of setting a title?

Non-blocking comment btw

**Review by ZJvandeWeg — APPROVED** [review]

**Review by Hasmin-AC — APPROVED** [review]

**Pull request merged**